unity-client/Assets/Tutorial/Scripts/Steps/TutorialStep_Tooltip.cs

```diff
@@ -50,6 +51,12 @@ public override IEnumerator OnStepExecute()
             yield return new WaitUntil(() => stepIsFinished);
         }
 
+        public override IEnumerator OnStepPlayAnimationForHidding()
```

Comment:
```suggestion
        public override IEnumerator OnStepPlayAnimationForHiding()
```
Or maybe:
```suggestion
        public override IEnumerator OnStepPlayHideAnimation()
```

Reply:
Ups! Sorry for my horrible english xD Anyway I liked more your second suggestion (`OnStepPlayHideAnimation`) and I have applied it. Done! Thanks! :)